UnityProject/Assets/Scripts/UI/Objects/Atmospherics/Pipes/GUI_Filter.cs

```diff
@@ -90,9 +76,9 @@ public void TogglePower()
 		}
 
 
-		public void SetMaxPressure(int To)
+		public void SetMaxPressure(int Value)
```

Comment:
```suggestion
		public void SetMaxPressure(int value)
```
I know it was ``To`` before, but we can amend those style errors now